### ISSUE TITLE
Access firebase database with a secret key

### ISSUE THREAD

**Issue by samitha9125** [invertase/react-native-firebase#1615]
Is there a way that we can use to access firebase realtime database with secret key when read is set to false?  I went through the document but couldn’t find anything related to that. Or any alternative to read firebase database data securely when read is set to false? 

Thanks.

Application Target Platform:
Andriod / iOS

Development Operating System:

Windows 10 and macOS 10.14 (Mojave)

Build Tools:
Basically use VSCode for coding and XCode and Android Studio for building apps.

React Native version:
0.55.4

React Native Firebase Version:

5.0.0

Firebase Module:
Auth

Are you using typescript?
No

**Comment by Salakar** [comment]
Please post questions on discord or StackOverflow, issues are purely for bug reports.

Only the Firebase Admin SDK on your backend can read when rules are set to false by using the service account.
